### PR TITLE
[FLINK-6956] Make table example POJO classes public

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

**Comment by fhueske** [comment]
+1 to merge

**Comment by greghogan** [comment]
Did we overlook adding a test to run the table examples? I think we do this for streaming and batch.

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/WordCountSQL.java [hunk 1]
```diff
@@ -65,7 +65,10 @@ public static void main(String[] args) throws Exception {
 	//     USER DATA TYPES
 	// *************************************************************************
 
-	private static class WC {
+	/**
+	 * Simple POJO containing a word and it's respective count.
```
Comment (greghogan) [inline flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/WordCountSQL.java]: "its" (and below)

**Comment by twalthr** [comment]
Yes, there are no tests, but the examples need to be reworked soon anyway. The Table API has much more features than shown in this very simple examples.

**Comment by zentol** [comment]
Will fix the typo while merging.

**Pull request closed without merging**